aries-site/src/themes/aries.js

```diff
@@ -7,6 +7,10 @@ export const aries = deepMerge(hpe, {
   // keeping file for use as playground for future theme adjusments that need
   // to be quickly tested
   pageHeader: {
```

Comment:
I think this can all be removed from aries.js now, correct?

Reply:
Done.